src/components/Button/button.types.d.ts

```diff
@@ -0,0 +1,22 @@
+import { HTMLAttributes } from 'react';
+
+/**
+ * Types for Button
+ * @param {primary} boolean - primary style of the button
+ * @param {secondary} boolean - secondary style of the button
+ */
+
+export interface IButtonProps extends HTMLAttributes<HTMLButtonElement> {
+  type?: 'primary' | 'secondary';
+}
+
+/**
+ * Types for Styled Button
+ * @param {primary} boolean - primary style of the button
+ * @param {secondary} boolean - secondary style of the button
+ */
+
+export interface IStyledButton {
+  primary?: boolean;
+  secondary?: boolean;
+}
```

Comment:
why do we need a separate interface for a styled button? Can it use the `IButtonProps` ?

Reply:
https://github.com/igorizviekov/mt-booking_app/pull/30/commits/44608ab65a6e86668930ffc2be6f022cec2a0cd8
https://github.com/igorizviekov/mt-booking_app/pull/30/commits/2f563bbead9993992fe9f7565954d4fd84e79f2b

Reply:
https://github.com/igorizviekov/mt-booking_app/pull/30/commits/44608ab65a6e86668930ffc2be6f022cec2a0cd8
https://github.com/igorizviekov/mt-booking_app/pull/30/commits/2f563bbead9993992fe9f7565954d4fd84e79f2b

Reply:
one more commit needed please =)